tests/scripts/get_list_for_release_note.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -e
+
+# NOTE: We can skip authenticating github username and token but that will limit the github api request to 60request/IP/hr and by authenticating the limit will be 5000.
+USERNAME=""
+TOKEN=""
+
+# This is just example branch name, change according to uses
+FROM_BRANCH="upstream/release-1.6" # example branch
+TO_BRANCH="v1.6.7" # example branch
+
+
+# for minor releases
+function minor_release() {
+for pr in $(git log --pretty="%s" --merges --left-only ${FROM_BRANCH}...${TO_BRANCH} | grep pull | sed -e "s/.*#\([0-9]\+\).*/\1/g" | sort -rn | uniq); do
```

Comment:
I made a few changes to the script to work for the v1.6.8 release. See [this commit](https://github.com/travisn/rook/commit/fd250822c744b9920f023498c896ad02c794437e).

Reply:
thanks for the changes @travisn , I think still we need one condition to check not sure that need but on manual backport we don't give original pr numbers like mergify does. This is mostly when we have conflicts or we do ci backport.

Reply:
With the manual backport PRs that don't match this pattern maybe we could just print the backport PR title, and we can manually fill in other details.

Reply:
right, I'll take your changes and modify and will add as co-author

Reply:
done